### ISSUE TITLE
Misleading output after 'Validate CSR' step

### ISSUE THREAD

**Issue by jlnhws** [diafygi/gethttpsforfree#26]
After completing the first three steps it says 'account registered...' after verifying the signatures. I'm stuck at this point. 'Step 4: Verify Ownership (waiting...)' hasn't changed since...

**Comment by licaon-kter** [comment]
So you've:
- created the folder _your-domain.abc/.well-known/acme-challenge_ ?
- created a file named _somethingsomethinglahblaha_ there as instructed?
- copy/paste the needed _strangegarbledtext_ inside the file as instructed?

The site error log has anything related?

**Comment by jlnhws** [comment]
No I haven't. If I understand correctly, the steps you pointed out are part of step 4, which isn't displaying any instructions on my end even though I completed steps 1 - 3.

**Comment by licaon-kter** [comment]
Try a clean profile browser, no extensions, no plugins.

A console log (right click Inspect->Console) would have been useful.

**Comment by jlnhws** [comment]
This is logged when I click the 'Validate Signatures' button: 

```
POST https://acme-v01.api.letsencrypt.org/acme/new-reg 409 (OK)
    validateInitialSigs @index.js:672
Uncaught TypeError: Cannot read property 'replace' of undefined
    requestChallenges @index.js:506
    account_xhr.onreadystatechange @index.js:663
```

Browser-info: Chromium Version 45.0.2454.85 Built on 8.1, running on Debian 8.2 (64-bit), incognito mode, no extensions, no plugins.

Update: Icweasel 38.2.1 logs the following: 

```
TypeError: d is undefined (index.js:506:12)
```

**Comment by jlnhws** [comment]
Apprently there was something wrong with my signing request. There weren't any domains. The output after Step 2's 'Validate CSR' is a bit misleading if you submit an invalid csr:

```
Found domains! Proceed to Step 3! ()
```

Only after looking at the source I understood that the domains are supposed to be printed in parenthesis. If there is no name the output should definitely reflect this, e.g.: "Error: Submitted CSR doesn't contain any valid domain names!"

**Comment by diafygi** [comment]
Thanks, fixed in 1c9c9b5
